### ISSUE TITLE
You need to add a Header file to build

### ISSUE THREAD

**Issue by Auase** [musescore/MuseScore#18058]
### Issue type

Other type of issue

### Bug description

You need to add # include "thirdparty/haw_logger/logger/log_base. h"  to 
 MuseScore\src\framework\audio\internal\audiobuffer.cpp to build, 
otherwise an error will be reported: audiobuffer. cpp: 165:9: No member named 'Helpful' in namespace 'how:: logger', 
but I am not sure if this will affect other code.

### Steps to reproduce

1.Resolve errors: audiobuffer.cpp:165:9: No member named 'Helpful' in namespace 'haw::logger'
2.add # include "thirdparty/haw_logger/logger/log_base. h"  to  MuseScore\src\framework\audio\internal\audiobuffer.cpp to build

### MuseScore Version

2023/6/20

### Regression

Choose option...

### Operating system

win11

**Comment by cbjeukendrup** [comment]
Something strange is going on here. The file `thirdparty/haw_logger/logger/log_base.h` _is_ already being included in `audiobuffer.h`, namely via `log.h`. Have you customised any build settings, for example have you disabled "unity build"?

**Comment by Auase** [comment]
I only added one line here: # include "thirdparty/haw_logger/logger/log_base. h" without modifying any other items in the project
![620](https://github.com/musescore/MuseScore/assets/56365735/4bcf43a7-31e6-4e12-a6a6-dab59bda3efe)

**Comment by Auase** [comment]
After obtaining the source code, I always run the default build, debugging, and deployment first, and only do other things after confirming that there are no issues

**Comment by Auase** [comment]
Is it misplaced?
![620_1](https://github.com/musescore/MuseScore/assets/56365735/147c99e4-62a4-4b25-9df5-c944b93de990)
![620_2](https://github.com/musescore/MuseScore/assets/56365735/00146b23-59f0-41f2-851c-665fb3a33b63)

**Comment by cbjeukendrup** [comment]
What do you mean? The code as on GitHub is correct; nobody else is experiencing these build issues, so it must be something in your configuration or your system. I wish I could say _what_, but unfortunately I have really no idea in this case.

**Comment by Auase** [comment]
Sorry, I misunderstood. Thirdparty/haw_ Logger/logger/log_ Base. h
It is indeed in MuseScore/src/framework/global/log. h.
I tried it again, it was strange that the first time I opened it, there was an error stopping the build and automatically locating it to audiobuffer CPP: 165:9 Here,
The second time I opened it, I deleted it
//#Include "thirdparty/haw_logger/logger/log_base. h"
Afterwards, it can be built again (unable to reproduce the first error. In fact, whether I change or not, audiobuffer. cpp: 165: error: No member named 'Helpful' in namespace 'how:: logger' exists).
Since it is not related here (but I cannot explain the strange automatic stop build behavior), this topic should be able to be closed,
I apologize again.
